### ISSUE TITLE
On CodeMirror, input fields, web standards and accessibility

### ISSUE THREAD

**Issue by sol** [swill/kad#36]
Hey there 👋

First things first, thanks for the tool. This really looks awesome, even though I wasn't able to fully capitalize on it.

/rant

TL;DR: Please don't use a `div`-tag (and by extension CodeMirror) as an input field.

The long story: This is not how the Internet works. Doing something like this is bound to break in certain situations. It is not good enough if it works on the two web browsers that somebody tested it on. Usually things break on less common system/browser combinations e.g. with Tridactyl on Firefox/Linux. However, in this particular case it breaks on iPads and possibly other iOS devices. Specifically copy & paste does not work. From what I understand, this basically renders the site unusable as you are supposed to copy & paste your layout from KLE.

Having a file upload feature, while still leaving the "input field" broken, would allow me to work around this issue.

/end rant
